CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 # IntelliJ Platform Plugin Template Changelog
 
 ## [Unreleased]
+### Added
+- Opt-out flag for bundling Kotlin standard library by default
```

Comment:
"Fix default to opt-out of bundling Kotlin standard library in plugin distribution"